### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Tracy-miranja**
- [ ] Add RSpec as a gem to your project.
- [ ] Create a Request spec file for all your controllers.
- [ ] Make sure that for each action you check:
- [ ] If response status was correct.
- [ ] If a correct template was rendered.
- [ ] If the response body includes correct placeholder text

.

### CODE REVIEW

File: spec/requests/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+RSpec.describe UsersController, type: :controller do
+  describe 'GET index' do
+    it 'assigns @users' do
+      user = User.create(name: 'John Doe')
+      get :index
+      expect(response).to be_successful
+      expect(assigns(:users)).to eq([user])
+    end
+  end
+end
```
Comment (Abenezer-Tilahun) [inline spec/requests/users_controller_spec.rb]: - [ ] As stated in the project requirement Kindly make sure that **If response status was correct**, **If a correct template was rendered** and **If the response body includes correct placeholder text.** So on above test case you check If the response body includes correct placeholder text So Kindly include the test case to check  **If response status was correct**, **If a correct template was rendered** Kindly just like the below sample 
```
 it 'responds with message success' do
      expect(response).to have_http_status(:ok)
    end

    it 'renders users list view' do
      expect(response).to render_template(:index)
    end
```
- [ ] Kindly I recommend you to include test case for **show actions** for User controllers as you write the test case for index actions Just like the below sample and kindly also include the above list three scenario

```
describe 'GET /show' do
  ...
end
 
```

File: spec/requests/posts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        expect(response).to render_template(:index)
+      end
+
+      it 'includes correct placeholder text in the response body' do
+        get posts_path
+        expect(response.body).to include('Display posts')
+      end
+    end
+  end
+end
```
Comment (Abenezer-Tilahun) [inline spec/requests/posts_controller_spec.rb]: - [ ] Kindly I recommend you to include test case for **show actions** for User controllers as you write the test case for index actions Just like the below sample and kindly also include the above list three scenario

```
describe 'GET /show' do
  ...
end
 
```

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Tracy-miranja  , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git -Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Tracy,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job implementing recommended changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
Kindly include instructions in your ReadMe on how to run the test cases of your project, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**